### ISSUE TITLE
15.0.5 Theming gone and admin theming options gone

### ISSUE THREAD

**Issue by dutchievj** [nextcloud-snap/nextcloud-snap#939]
Yesterday I created a new server and putted the NC snap export backup back in.

To my surprise the login window background etc and everything else is back to default.

I logged in as an admin and the options to edit the colors and background of the login window are gone.

Did I do something wrong while exporting -> importing it all? All but the theming options did not work.

**Comment by kyrofa** [comment]
The theming app is disabled by default in v15. I would have expected the fact that it was enabled to be included in your export, but for experimentation's sake, what happens if you enable it? Does everything look correct?

**Comment by dutchievj** [comment]
Everything looks correct!

I enabled the disabled Theme app and it showed up under admin settings. There I found my old theme still in tact. The moment I enabled the Theme app all started to look how I left it.

**Comment by kyrofa** [comment]
Alright, walk me through how you did this. We still ship the app obviously, we just disable it once you create an admin user (it's impossible to do before then). Did you import before doing that?

**Comment by dutchievj** [comment]
The admin user was in my backup so when I imported my backup I had an admin user.
On the new server I installed the nextcloud snap. 
Directly after that I restored the backup. All worked and was restored but no theming. 
Enabled the theming app under the admin account apps and everything was like on the old server. Look and all and login window etc was like before the moment I enabled the theming app.
